Last login: Thu Oct  5 08:21:57 on ttys000
Ninas-MacBook-Pro:~ Niner$ source flaskEnv/bin/activate
-bash: flaskEnv/bin/activate: No such file or directory
Ninas-MacBook-Pro:~ Niner$ cd Desktop
Ninas-MacBook-Pro:Desktop Niner$ cd DojoAssignments
Ninas-MacBook-Pro:DojoAssignments Niner$ ls
FlaskFundamentals	MyEnvironments		WebFundamentals
Icon?			Python			iOS
MEAN			Ruby			netCore
Ninas-MacBook-Pro:DojoAssignments Niner$ cd MyEnvironments
Ninas-MacBook-Pro:MyEnvironments Niner$ virtualenv flaskEnv
New python executable in /Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/bin/python
Installing setuptools, pip, wheel...done.
Overwriting /Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/bin/activate with new content
Overwriting /Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/bin/activate.fish with new content
Overwriting /Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/bin/activate.csh with new content
Overwriting /Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/bin/python-config with new content
Ninas-MacBook-Pro:MyEnvironments Niner$ source flaskEnv/bin/activate
(flaskEnv) Ninas-MacBook-Pro:MyEnvironments Niner$ pip install flask
Requirement already satisfied: flask in ./flaskEnv/lib/python2.7/site-packages
Requirement already satisfied: itsdangerous>=0.21 in ./flaskEnv/lib/python2.7/site-packages (from flask)
Requirement already satisfied: Werkzeug>=0.7 in ./flaskEnv/lib/python2.7/site-packages (from flask)
Requirement already satisfied: Jinja2>=2.4 in ./flaskEnv/lib/python2.7/site-packages (from flask)
Requirement already satisfied: click>=2.0 in ./flaskEnv/lib/python2.7/site-packages (from flask)
Requirement already satisfied: MarkupSafe>=0.23 in ./flaskEnv/lib/python2.7/site-packages (from Jinja2>=2.4->flask)
(flaskEnv) Ninas-MacBook-Pro:MyEnvironments Niner$ cd..
-bash: cd..: command not found
(flaskEnv) Ninas-MacBook-Pro:MyEnvironments Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:DojoAssignments Niner$ cd flaskfundamentals
(flaskEnv) Ninas-MacBook-Pro:flaskfundamentals Niner$ cd helloflask
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 10:49:06] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/FlaskFundamentals/HelloFlask/hello.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/FlaskFundamentals/HelloFlask/hello.py", line 14, in success
    return render_template('success.html')
NameError: global name 'render_template' is not defined
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:49:21] "GET /success?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/FlaskFundamentals/HelloFlask/hello.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "hello.py", line 1, in <module>
    from flask import Flask, render_templates
ImportError: cannot import name render_templates
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 10:51:54] "GET /success HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/FlaskFundamentals/HelloFlask/hello.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "hello.py", line 14, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 10:54:18] "GET /index HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:55:25] "GET /index HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 10:59:33] "GET /success HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/FlaskFundamentals/HelloFlask/hello.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "hello.py", line 18, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
Traceback (most recent call last):
  File "hello.py", line 18, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
Traceback (most recent call last):
  File "hello.py", line 18, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ ls
hello.py	templates
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
Traceback (most recent call last):
  File "hello.py", line 18, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
Traceback (most recent call last):
  File "hello.py", line 18, in <module>
    @app.route('/success')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: success
(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ python hello.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:05:12] "GET /success HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloFlask/hello.py", line 16, in success
    return render_template('success.html')
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 57, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 85, in _get_source_fast
    raise TemplateNotFound(template)
TemplateNotFound: success.html
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:20:58] "GET /success?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
ls  
^C(flaskEnv) Ninas-MacBook-Pro:helloflask Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd FlaskFundamentals
-bash: cd: FlaskFundamentals: No such file or directory
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd HelloFlask
(flaskEnv) Ninas-MacBook-Pro:HelloFlask Niner$ python hello.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:23:19] "GET /success HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:23:28] "GET /success HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:23:52] "GET / HTTP/1.1" 200 -
^C(flaskEnv) Ninas-MacBook-Pro:HelloFlask Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd FlaskFundamentals
-bash: cd: FlaskFundamentals: No such file or directory
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ ls
HelloFlask	HelloWorld
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd HelloWorld
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import flask, reader_template
ImportError: cannot import name flask
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import flask, reader_template
ImportError: cannot import name flask
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ cd templates
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch index.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from Flask import Flask, reader_template
ImportError: No module named Flask
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from Flask import Flask, reader_template
ImportError: No module named Flask
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ ls
HelloWorld.py	templates
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
Traceback (most recent call last):
  File "HelloWorld.py", line 1, in <module>
    from flask import Flask, reader_template
ImportError: cannot import name reader_template
(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ python HelloWorld.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:44:57] "GET / HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:45:06] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:45:43] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
TypeError: index() takes exactly 1 argument (0 given)
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:17] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
TypeError: index() takes exactly 1 argument (0 given)
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 11:47:27] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:48:04] "GET /index HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:49:55] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 11:50:08] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/HelloWorld/HelloWorld.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
^C(flaskEnv) Ninas-MacBook-Pro:HelloWorld Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd Portfolio
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ touch portfolio.py
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ mkdir templates
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ cd templates
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch index.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch projects.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch about.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ ls
portfolio.py	templates
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ python portfolio.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:05:06] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:05:24] "GET /projects HTTP/1.1" 404 -
cd .. 
c^C(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd Portfolio
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ ls
portfolio.py	templates
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ python portfolio.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:06:01] "GET /projects HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 12:06:05] "GET /projects HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 12:06:07] "GET /projects HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 12:06:10] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/Portfolio/portfolio.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
ls
^C(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ ls
portfolio.py	templates
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ python portfolio.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:09:47] "GET / HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/Portfolio/portfolio.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:10:14] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 12:10:15] "GET / HTTP/1.1" 404 -
python portfolio.py
^C(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ python portfolio.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:10:42] "GET / HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/Portfolio/portfolio.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:11:26] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:11:33] "GET /about HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:11:44] "GET /about HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:11:50] "GET /projects HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:15:02] "GET /about HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:15:14] "GET /about HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:15:30] "GET /about HTTP/1.1" 200 -
^C(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ ls
portfolio.py	templates
(flaskEnv) Ninas-MacBook-Pro:Portfolio Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ mkdir FormTest
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd FormTest
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ touch server.py
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ mkdir templates
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ cd templates
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch index.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch style.css
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ ls
server.py	templates
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:24:36] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 12:24:51] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:24:51] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 12:24:52] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:24:52] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 12:24:53] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:24:53] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 12:24:53] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:24:53] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "server.py", line 23, in <module>
    @app.route('/users', methods=['POST'])
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: create_user
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
Traceback (most recent call last):
  File "server.py", line 23, in <module>
    @app.route('/users', methods=['POST'])
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1051, in add_url_rule
    'existing endpoint function: %s' % endpoint)
AssertionError: View function mapping is overwriting an existing endpoint function: create_user
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
  File "server.py", line 16
    print request.form
                     ^
IndentationError: unindent does not match any outer indentation level
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python servor.py
python: can't open file 'servor.py': [Errno 2] No such file or directory
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FlaskFundamentals Niner$ cd FormTest
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ ls
server.py	templates
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:41:53] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:42:22] "GET / HTTP/1.1" 200 -
Got Post Info
ImmutableMultiDict([('name', u'hey '), ('email', u'yo')])
127.0.0.1 - - [05/Oct/2017 12:42:28] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:42:28] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:43:26] "GET / HTTP/1.1" 200 -
Got Post Info
ImmutableMultiDict([('name', u'hey'), ('email', u'jkbk')])
127.0.0.1 - - [05/Oct/2017 12:43:30] "POST /users HTTP/1.1" 200 -
^C(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ ls
server.py	templates
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:47:43] "GET /users HTTP/1.1" 405 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:47:55] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:47:56] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:47:57] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:47:57] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:47:57] "GET /users HTTP/1.1" 405 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:48:07] "GET /users HTTP/1.1" 405 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:48:41] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:48:42] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:48:42] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:48:42] "GET /users HTTP/1.1" 405 -
127.0.0.1 - - [05/Oct/2017 12:48:46] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 12:48:50] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 12:48:50] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
Traceback (most recent call last):
  File "server.py", line 17, in <module>
    request.form['name']
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/globals.py", line 37, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ cd templates
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ touch success.html
(flaskEnv) Ninas-MacBook-Pro:templates Niner$ cd ..
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:51:51] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:51:54] "POST /users HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:52:17] "POST /users HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 12:52:21] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:52:24] "POST /users HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:52:56] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:53:00] "POST /users HTTP/1.1" 404 -
^C(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:53:18] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 12:53:21] "POST /users HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 12:59:21] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 15
    return render_template("index.html", name="Jay")    #mybe redirect?
SyntaxError: 'return' outside function
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
  File "server.py", line 15
    return render_template("index.html", name="Jay")    #mybe redirect?
SyntaxError: 'return' outside function
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:23:20] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:23:22] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:23:22] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:23:22] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:25:01] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:27:29] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:28:23] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 7
    return render_templa te("index.html", phrase='Hello', times= 5)
                          ^
SyntaxError: invalid syntax
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
  File "server.py", line 7
    return render_templa te("index.html", phrase='Hello', times= 5)
                          ^
SyntaxError: invalid syntax
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    {% for x in range(0,5) %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:30:52] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    {% for x in range(0,5) %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:11] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    {% for x in range(0,5) %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:12] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    {% for x in range(0,5) %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:29] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    {% for x in range(0,5) %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:30] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:47] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    <!-- {% for x in range(0,5) %} -->
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:47] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:47] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:47] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:48] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:48] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:48] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:51] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 7, in index
    return render_template("index.html", phrase='Hello', times= 5)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 11, in template
    <!-- {% for x in range(0,5) %} -->
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
127.0.0.1 - - [05/Oct/2017 13:31:51] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:51] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:51] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:52] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:52] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:31:52] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:32:22] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:36:50] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:38:12] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 15
    return render_template("index.html", name="Jay")    #mybe redirect?
SyntaxError: 'return' outside function
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
  File "server.py", line 15
    return render_template("index.html", name="Jay")    #mybe redirect?
SyntaxError: 'return' outside function
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:45:10] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:45:13] "POST /users HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 15, in create_user
    return render_template("index.html", name="Jay")    #mybe redirect?
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 14, in top-level template code
    {% for k in range(0,times) %}
UndefinedError: 'times' is undefined
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:45:13] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:46:12] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:46:15] "POST /users HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 15, in create_user
    return render_template("index.html", name="Jay")
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/index.html", line 14, in top-level template code
    {% for k in range(0,times) %}
UndefinedError: 'times' is undefined
127.0.0.1 - - [05/Oct/2017 13:46:15] "GET /users?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:46:15] "GET /users?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:46:15] "GET /users?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:46:15] "GET /users?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:46:15] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:46:47] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:47:09] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:47:13] "POST /users HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:47:16] "POST /users HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:49:27] "POST /users HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 15, in create_user
    return render_template("success.html", name="Jay")
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/success.html", line 17, in top-level template code
    {% for k in range(0,times) %}
UndefinedError: 'times' is undefined
127.0.0.1 - - [05/Oct/2017 13:49:27] "GET /users?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:27] "GET /users?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:27] "GET /users?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:27] "GET /users?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:27] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:28] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:36] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:49:40] "POST /users HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py", line 15, in create_user
    return render_template("success.html", name="Jay")
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/Niner/Desktop/DojoAssignments/MyEnvironments/flaskEnv/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/templates/success.html", line 17, in top-level template code
    {% for k in range(0,times) %}
UndefinedError: 'times' is undefined
127.0.0.1 - - [05/Oct/2017 13:49:40] "GET /users?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:40] "GET /users?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:40] "GET /users?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:40] "GET /users?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:40] "GET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:49:53] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Oct/2017 13:49:56] "POST /users HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:50:54] "POST /users HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:52:06] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 13:52:06] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:52:08] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 13:52:09] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:52:27] "GET / HTTP/1.1" 200 -
Got Post Info
127.0.0.1 - - [05/Oct/2017 13:52:30] "POST /users HTTP/1.1" 302 -
127.0.0.1 - - [05/Oct/2017 13:52:30] "GET /success HTTP/1.1" 404 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:52:34] "GET /success HTTP/1.1" 404 -
127.0.0.1 - - [05/Oct/2017 13:52:40] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 10
    return render_template("success.html", phrase='Hello', times= 5)
    ^
IndentationError: unexpected indent
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
127.0.0.1 - - [05/Oct/2017 13:54:15] "GET / HTTP/1.1" 200 -
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 10
    return render_template("success.html", phrase='Hello', times= 5)
    ^
IndentationError: unexpected indent
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python sercer.py
python: can't open file 'sercer.py': [Errno 2] No such file or directory
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 586-042-235
 * Detected change in '/Users/Niner/Desktop/DojoAssignments/Python/FlaskFundamentals/FormTest/server.py', reloading
 * Restarting with stat
  File "server.py", line 17
    def create_success():
    ^
IndentationError: unexpected indent
(flaskEnv) Ninas-MacBook-Pro:FormTest Niner$ python server.py
